src/app/app.module.ts

```diff
@@ -1,31 +1,41 @@
+import { HttpModule } from '@angular/http';
+import { RouterModule } from '@angular/router';
+
 import { BrowserModule } from '@angular/platform-browser';
 import { ErrorHandler, NgModule } from '@angular/core';
 import { IonicApp, IonicErrorHandler, IonicModule } from 'ionic-angular';
 
 import { MyApp } from './app.component';
 import { HomePage } from '../pages/home/home';
+import { ArticlePage } from '../pages/article/article';
 
 import { StatusBar } from '@ionic-native/status-bar';
 import { SplashScreen } from '@ionic-native/splash-screen';
+import { ArticlesDataProvider } from '../providers/articles-data/articles-data';
 
 @NgModule({
   declarations: [
     MyApp,
-    HomePage
+    HomePage,
+    ArticlePage
```

Comment:
remove all mentions of the article page in this file

Reply:
Done